### PR TITLE
Feature/windows Version

### PR DESCRIPTION
**Pull request by fbrissi**
#42 

- Added Inno Setup config to install for windows
- When run in windows `KOOL_ASUSER` set 0

### CODE REVIEW

**Comment by dbpolito** [comment]
❤️ This is looking great.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ In case you need sudo:
...
 ```
 
+For Windows
+
+Download the installer [here](https://github.com/kool-dev/kool/releases)
+
+Or execute the bat with admin
+
+```bash
+bitsadmin /transfer KoolDevInstallDownload /dynamic /download /priority foreground https://raw.githubusercontent.com/kool-dev/kool/master/install.bat install-kool-dev.bat
```
Comment (fabriciojs) [inline README.md:45]: Is this bat helpful/necessary at some situation? I mean, having the installer seems like the best/simpler option for Windows native users.

File: enviroment/auser.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// +build !windows
```
Comment (fabriciojs) [inline enviroment/auser.go:1]: I guess the file name was intended to be `asuser.go`?

File: inno-setup/kool.iss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+SetupIconFile={#ApplicationName}.ico
+UninstallDisplayIcon={autopf}\{#ApplicationGroup}\{#ApplicationName}.ico
+UninstallDisplayName={#ApplicationName}
+WizardImageStretch=No
+WizardSmallImageFile={#ApplicationName}-setup-icon.bmp
+ArchitecturesInstallIn64BitMode=x64
+ChangesEnvironment=true
+
+[Languages]
+Name: pt_BR; MessagesFile: "compiler:Languages\BrazilianPortuguese.isl"
```
Comment (fabriciojs) [inline inno-setup/kool.iss:39]: I guess we only have support for English, so would make sense to remove pt_BR?

File: install.bat [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+if "%PROCESSOR_ARCHITECTURE%" == "x86" (set DEFAULT_BIN_PATH="%ProgramFiles(x86)%\kool-dev") else (set DEFAULT_BIN_PATH="%ProgramFiles%\kool-dev")
+DEFAULT_BIN=%DEFAULT_BIN_PATH%
+if "%BIN_PATH%" == "" set BIN_PATH=%DEFAULT_BIN%
+
+:command_exists
+	call -v %* > NULL
+
+:do_install
+	if "%PROCESSOR_ARCHITECTURE%" == "x86" (set ARCH=386) else (set ARCH=amd64)
```
Comment (fabriciojs) [inline install.bat:18]: this is allowing the script to consider arch `386` which we are not building it... is this Bar necessary with the installer? can we drop it?

**Review by fabriciojs — CHANGES_REQUESTED** [review]
Awesome! I just left a few comments, and also would like to note that we should add some entry on the documentation markdown files about Windows support.

File: README.md (hunk 1 above)
Comment (fbrissi) [reply README.md:45]: No more, the problem with this is the way, in each new installation the path is added, and is not easy to uninstall.

The installer solve this problem.

File: install.bat (hunk 4 above)
Comment (fbrissi) [reply install.bat:18]: This file is no longer needed. I will remove it.

File: enviroment/auser.go (hunk 2 above)
Comment (fbrissi) [reply enviroment/auser.go:1]: Yes, you are right.

Comment (danielsuguimoto) [reply enviroment/auser.go:1]: @fbrissi just an observation: `asuser_windows.go` file has the same typo issue. And the function `InitAsUser' is not written correctly on both files.

File: build_artifacts.sh [hunk 5]
```diff
@@ -25,6 +25,15 @@ echo "Building to GOOS=linux GOARCH=arm GOARM=7"
 
 docker run --rm --env GOOS=linux --env GOARCH=arm --env GOARM=7 --env CGO_ENABLED=0 -v $(pwd):/code -w /code $GO_IMAGE go build -a -tags 'osusergo netgo static_build' -ldflags '-extldflags "-static"' -o dist/kool-linux-arm7
 
+echo "Building to GOOS=windows GOARCH=amd64"
+
+docker run --rm --env GOOS=windows --env GOARCH=amd64 --env CGO_ENABLED=0 -v $(pwd):/code -w /code $GO_IMAGE go build -a -tags 'osusergo netgo static_build' -ldflags '-extldflags "-static"' -o dist/kool.exe
+
+echo "Building kool-install.exe"
+
+docker run --rm -i -v $(pwd):/work amake/innosetup /dApplicationVersion=1.0.15 inno-setup/kool.iss
```
Comment (fbrissi) [inline build_artifacts.sh:34]: @fabriciojs any suggestions for version?

**Comment by fbrissi** [comment]
### Today I did many tests on a project with windows, It's ok for me.

WhatsApp Image 2020-09-03 at 20 31 14
WhatsApp Image 2020-09-03 at 20 30 50
WhatsApp Image 2020-09-03 at 20 29 35
WhatsApp Image 2020-09-03 at 20 28 59
WhatsApp Image 2020-09-03 at 20 28 38
WhatsApp Image 2020-09-03 at 20 27 59
WhatsApp Image 2020-09-03 at 20 27 21
image

File: build_artifacts.sh (hunk 5 above)
Comment (fabriciojs) [reply build_artifacts.sh:34]: That is OK for now. We do have an open issue for dealing with hard-coded versions, we can deal with that later.

**Review by fabriciojs — APPROVED** [review]

**Review by danielsuguimoto — APPROVED** [review]

**Pull request merged**